charts/trino/templates/configmap-coordinator.yaml

```diff
@@ -35,10 +41,18 @@ data:
     http-server.http.port={{ .Values.service.port }}
     query.max-memory={{ .Values.server.config.query.maxMemory }}
     query.max-memory-per-node={{ .Values.server.config.query.maxMemoryPerNode }}
+    query.max-total-memory-per-node={{ .Values.server.config.query.maxTotalMemoryPerNode }}
+    memory.heap-headroom-per-node={{ .Values.server.config.memory.heapHeadroomPerNode }}
     discovery-server.enabled=true
     discovery.uri=http://localhost:{{ .Values.service.port }}
+  {{- range $configValue := .Values.additionalConfigProperties }}
+    {{ $configValue }}
+  {{- end }}
 
   log.properties: |
     io.trinosql={{ .Values.server.log.trino.level }}
```

Comment:
This should be `io.trino`. There is no logger under `io.trinosql`.

Please verify the chart for other occurences of `trinosql` to make sure there are no leftovers from the rename.